frontend/src/components/toast/toast-pie-chart.js

```diff
@@ -0,0 +1,115 @@
+import React from "react";
+import Style from "style-it";
+
+import { PieChart, Pie, Legend } from "recharts";
+
+import { numWithCommas } from "../../utils/plan-utils";
+
+class ToastPieChart extends React.Component {
+  constructor(props) {
+    super(props);
+
+    this.getLabel = this.getLabel.bind(this);
+  }
+
+  getLabel(e) {
+    return (
+      <text x={e.x - 1.5} y={e.y} textAnchor="middle">
+        {`$${numWithCommas(e.value)}`}
+      </text>
+    );
+  }
+
+  render() {
+    if (this.props.subheader) {
+      var subheader = "/$" + numWithCommas(this.props.subheader);
+    }
+    const styles = `
+        .hidden {
+            visibility: hidden;
+            max-height: 0px;
+        }
+    
+        .caption {
+            padding: 0 3rem 0 3rem;
+        }
+    
+        .chart {
+            display: flex;
+            align-items: center;
+            flex-direction: column;
+            background-color: var(--toast-neutral-5);
+            padding: 2rem 5rem 5rem 5rem;
+            margin: 0 3rem 3rem 3rem;
+            border-radius: 0.5rem;
+        }
+    
+        .top-right {
+            align-self: flex-end;
+            padding-bottom: 4rem;
+            display: flex;
+            align-items: flex-end;
+        }
+    
+
+        .subheader {
+            margin-top: 0.5rem;
+            margin-left: 0.25rem;
+        }
+        .recharts-sector{
+            stroke:none;
+        }
+
+        .recharts-pie-label-line{
+            d:none;
+        }
+        `;
+    return Style.it(
+      `${styles}`,
+      <div>
+        <svg className="hidden">
+          <linearGradient id="gradient" x1="0%" y1="0%" x2="100%" y2="0%">
+            <stop
+              offset="0%"
+              stopColor="var(--toast-purple-2)"
+              stopOpacity="1"
+            />
+            <stop
+              offset="100%"
+              stopColor="var(--toast-blue-2)"
+              stopOpacity="1"
+            />
+          </linearGradient>
+        </svg>
+
+        <div className="chart">
+          <div className="row">
+            <div className="top-right">
```

Comment:
this isn't showing up on the top right for me, it's showing up in the center. would be good if you could also put the toggle in here so we can see if the component is the only thing that's pushing the page down